newsfragments/2686.misc.rst

```diff
@@ -0,0 +1 @@
+Ursula will now detect absence of active stakes during startup and in worker loop.
```

Comment:
Is this the correct newsfragment for this PR?

Reply:
Bump - I see the edge case handling, but not the check for active stakes.  Am I missing it right in front of me?

Reply:
No, this newsfragment is entirely incorrect and was meant for a different PR. That was a blunder.